.changeset/beige-peas-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/addons': minor
```

Comment:
sorry! `addons` is a private package so we'll need to increment `sv` instead, which is also in pre-release so it should also be a `patch`
```suggestion
'sv': patch
```